tasks/broad/ImputationTasks.wdl

```diff
@@ -1164,185 +1007,3 @@ task CreateVcfIndex {
     File vcf_index = "~{vcf_basename}.tbi"
   }
 }
-
-task PreSplitVcf {
```

Comment:
just for my understanding, PreSplitVcf and PreChunkVcf were tasks we'd added to make this work nice on Azure, where we couldn't stream, and we've subsequently removed those tasks from the WDL so we can delete them here entirely? should we go ahead and delete [ImputationBeaglePreChunk.wdl](https://github.com/broadinstitute/warp/blob/TSPS-183_mma_beagle_imputation_hg38/pipelines/broad/arrays/imputation_beagle/ImputationBeaglePreChunk.wdl) since I think that does use these tasks?

Reply:
oh i totally forgot that wdl even existed.  I dont think we're ever gonna need that version and by the time we do we should have streaming so totally ok getting rid of that wdl